src/library/scala/deprecated.scala

```diff
@@ -58,4 +58,4 @@ import scala.annotation.meta._
  *  @see    [[scala.deprecatedName]]
  */
 @getter @setter @beanGetter @beanSetter
-final class deprecated(message: String = "", since: String = "") extends scala.annotation.StaticAnnotation
+class deprecated(message: String = "", since: String = "") extends scala.annotation.StaticAnnotation
```

Comment:
or make it `sealed class deprecated` and introduce `unused` like thing?

Reply:
You're just going to end up rehashing the conversation in https://github.com/scala/bug/issues/10790

I don't understand the motivation to make the annotations `final`. It achieves very little, and stops people from creating convenient default parameters, like `@unused`, which is the primary use-case for inheriting from an annotation.

Any change to the compiler or stdlib is going to break something for somebody, so it needs to have a strong justification. I believe there is a design document explaining the process for changing the public API, and the original PR should have been subjected to those rules. I don't know why this change skipped that process.

Reply:
I still don't understand why extending annotations is a problem. I also don't understand why the current situation even needs a "better design". Scala embraces subtyping, this is a great example of subtyping being deployed for a practical use.

Reply:
`Scala embraces subtyping` not means you should use it everywhere.
If you want to make use of one battery packaged annotation, then the extending won't help.
A better thing is introducing the Java annotation mechanism which embraces composition. 

Another thing is the current non-final thing is a kind of leak, if they were designed to be inherited, then they should be very documented but not a gist posted here and there.

Reply:
making this `final` is arbitrary and breaks existing codebases for no gains. I have nothing more to say on the matter.